frontend/ui/elements/screensaver_menu.lua

```diff
@@ -38,7 +38,7 @@ return {
             genMenuItem(_("Leave screen as-is"), "screensaver_type", "disable", nil, true),
             separator = true,
             {
-                text = _("Border fill"),
+                text = _("Border fill and fit"),
```

Comment:
but where are the screenshots. you are running a marathon but stopping short of the finishing line.

Reply:
Rotation off:
![v-rnor](https://github.com/user-attachments/assets/3ccbbe2e-241f-4db1-a816-e0d8f52d915c)

Rotation on:
![v-rnor](https://github.com/user-attachments/assets/94bdd1aa-6cb6-4f17-9db8-0754ff896f49)

Rotation off:
![h--](https://github.com/user-attachments/assets/b9782210-caa1-4b00-a790-5fbcdf2af207)

Rotation on:
![h-r](https://github.com/user-attachments/assets/e1054e74-ed69-42ba-aeb5-117b6e9f25ac)

Rotation and stretch on:
![hsr](https://github.com/user-attachments/assets/19e7fa34-7716-41af-9edd-71d3779343fd)

If you're not satisfied with that, you can pull my branch and run your own tests.

Reply:
stop marking it as resolved so other can see them too.

yes, I can see how you setting should live next to the stretch one. However I am not convinced now that the wording for said setting is appropriate `"Rotate cover to fit screen`. As far as I see, you just rotate the image and then stretch with the existing stretch-setting.

Reply:
I'll unresolve since you raised a new issue.

True, it's not clear what the setting does from the name. How about the same language as the image viewer setting: "Auto-rotate cover for best fit"

Reply:
> True, it's not clear what the setting does from the name. How about the same language as the image viewer setting: "Auto-rotate cover for best fit"

Yup, I seem to remember there was already a lot of back-and-forth on that one so let's avoid any more useless bike-shedding on that front ;).

Reply:
👍

Reply:
I can live with “Auto-rotate cover for best fit“ and “Border fill, fit and rotation”. But lets see if anyone comes up with anything else.

Reply:
> However I am not convinced now that the wording for said setting is appropriate `"Rotate cover to fit screen`. As far as I see, you just rotate the image and then stretch with the existing stretch-setting. @Frenzie

Sounds like excellent, clear wording to me. Could you explain in what way this could be construed as confusing?

Reply:
Because according to that wording both image 3 and 4 technically satisfy the criteria of the string. They both "fit" by using the maximum height or width of the  screen. Furthermore since the rotation does not always happen, one might think said setting doesn't really work, if like me, you always have sane covers that wouldn't be rotated at all with this. "Best fit" however i am not overly fond of but at least sort of hints at checking which of the two options would use a larger area of the display removing the example 3.

Reply:
Well of course. 3 fits properly, 4 fills up all the space as per the stretch setting. But that's not the part of the string I'm concerned about. It's the "auto" that's an eyesore.

Reply:
What's wrong with it? Even Apple doesn't mind it ![image](https://github.com/user-attachments/assets/591cd69c-e492-494d-aa17-fcd7ad650052)

Reply:
While indeed hideous, that's completely different. "Autocapitalize" is the same thing as "Capitalize first letter" in the same way that "Autorotate" is the same thing as "Rotate for best fit."

Reply:
Okay, "rotate for best fit" and "border fill, rotation and fit" are fine.

Reply:
I'm happy with that. You could also turn it around: `Best-fit rotation`.

But perhaps that would imply too much…
https://matthias-research.github.io/pages/publications/stablePolarDecompSlides.pdf
![image](https://github.com/user-attachments/assets/113862d9-7d2e-4736-8b99-9bd988acbc29)

Reply:
I'm not willing to devote more attention to this PR. I appreciate the suggestions and discussions that were proportionate to understanding, but if anyone wants to develop it further, please submit a new PR :)

Reply:
People are very passionate about their reading experience around here, and anything that can improve said experience is always welcome, even if sometimes things can get a bit frisky ;). Having said that, I don't think anyone expects you to implement that "Robust Method to Extract the Rotational Part of Deformations". Common, you are almost there.

And btw, than you for implementing this. I'm sure plenty of folks will be glad to finally be able to display landscape-shot pics of their feline friends on their screens.

Reply:
```suggestion
                text = _("Border fill, rotation, and fit"),
```